package-lock.json

```diff
@@ -1,16530 +1,8 @@
 {
   "name": "vue-plausible",
   "version": "1.1.2",
-  "lockfileVersion": 2,
```

Comment:
Seems like we might be using a different version of npm, which probably caused the change in `lockfileVersion` and the large amount of deletions.

Reply:
Sorry about that. I pushed an update with `npm` `7.5.4`. The line count for `package-lock.json` is down to only 20 lines. It still includes `defu` but only for development (probably for another dependency). Let me know if there is anything else :confetti_ball: